acestep/core/generation/handler/lyric_timestamp.py

```
@@ -0,0 +1,120 @@
+                calc_matrix=align_info["calc_matrix"],
+                lyrics_tokens=pure_lyric_ids,
+                total_duration_seconds=total_duration_seconds,
+            )
+            return {
+                "lrc_text": result["lrc_text"],
+                "sentence_timestamps": result["sentence_timestamps"],
+                "token_timestamps": result["token_timestamps"],
+                "success": True,
+                "error": None,
+            }
+        except Exception as exc:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)